src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/PricingType.php

```diff
@@ -95,6 +121,32 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'label_tag_name' => 'h2',
                 'label_help_box' => $this->trans('Set specific prices for customers meeting certain conditions.', 'Admin.Catalog.Help'),
             ])
+            ->add('show_catalog_price_rules', IconButtonType::class, [
+                'icon' => 'visibility',
+                'label' => $this->trans('Show catalog price rules', 'Admin.Catalog.Feature'),
+                'attr' => [
+                    'data-hide-label' => $this->trans('Hide catalog price rules', 'Admin.Catalog.Feature'),
+                    'data-show-label' => $this->trans('Show catalog price rules', 'Admin.Catalog.Feature'),
+                    'data-show-icon' => 'visibility',
+                    'data-hide-icon' => 'visibility_off',
```

Comment:
Maybe those two can be specified, I mean since it's not reused and doesn't need to be configured you can hard code those values in the js It removes two data attributes here, and two variables in the JS code